pkg/config/auth.go

```diff
@@ -18,6 +18,8 @@ type (
 		SiteUrl                    string   `toml:"site_url"`
 		AdditionalRedirectUrls     []string `toml:"additional_redirect_urls"`
 		JwtExpiry                  uint     `toml:"jwt_expiry"`
+		MinimumPasswordLength      uint     `toml:"minimum_password_length"`
+		PasswordRequirements       string   `toml:"password_requirements"`
```

Comment:
Since we started supporting `config push` command, adding new auth config also requires mapping to the api request / response fields https://github.com/supabase/cli/blob/develop/pkg/config/auth.go#L186.

Once that's mapped, remember to update the diff snapshots in `pkg/config/testdata` to resolve the failing tests.

Reply:
@sweatybridge I think this is going to be an issue for me as I don't know what these keys are on the hosted platform.

Reply:
No worries, I can take over from here next week.

Fyi, all fields are more or less documented in the [openapi spec](https://github.com/supabase/cli/blob/develop/pkg/api/types.gen.go#L465).

Reply:
I figured it out. The only part is the `testdata` I'm having trouble with now.

Reply:
@sweatybridge Are the diff files manually created or is there a command to run to generate them?

Reply:
Some tests don't work in WSL due to the reliance on keyring which is not available in WSL. This means I can't run the test suite from my end.

![image](https://github.com/user-attachments/assets/744f43e2-6838-45df-8e18-13ef7bc3fdd5)

Reply:
To run a single test case:

```
go test -v ./pkg/config -run "TestHookDiff/local_and_remote_enabled"
```

Reply:
@sweatybridge I've tried rerunning the corresponding test but it isn't generating any new diff file.

Reply:
And you've deleted the diff file before running the single test? Could be a windows issue then 😅

Reply:
OK got this working now. I used the run test functionality of VSCode instead because the go test command didn't seem like it was creating new diff files.